sdk/servicebus/azure-servicebus/tests/test_queues.py

```diff
@@ -1251,7 +1251,7 @@ def test_queue_by_servicebus_client_renew_message_locks(self, uamqp_transport, *
                     receiver.complete_message(messages[1])
                     assert (messages[2].locked_until_utc - utc_now()) <= timedelta(seconds=60)
                     sleep_until_expired(messages[2])
-                    with pytest.raises(ServiceBusError):
+                    with pytest.raises(MessageLockLostError):
```

Comment:
In theory, all of these tests should run successfully without any modifications, given that this is still a ServiceBusError. Did we ever try that?

Reply:
I tried it locally and it worked, but will run tests on only code changes without changing the test suite just to make sure + link the run here.

Reply:
https://github.com/Azure/azure-sdk-for-python/pull/30971

Reply:
all tests passed on run here: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=2886363&view=results